packages/lesswrong/components/walledGarden/PortalBarGcalEventItem.tsx

```diff
@@ -27,8 +27,8 @@ export const getAddToCalendarLink = (gcalEvent) => {
   
   const UrlClass = getUrlClass()
   const url = new UrlClass(gcalEvent.htmlLink)
-  const eid = url.searchParams.get("eid")
-  const addToCalendarLink = `https://calendar.google.com/event?action=TEMPLATE&tmeid=${eid}&tmsrc=${gcalEvent.organizer.email}`
+  const eid = sanitize(url.searchParams.get("eid"))
```

Comment:
This is not the sanitization this variable needs. This variable needs URI encoding. The sanitize function sanitizes HTML inputs. Also, you can't use the sanitize function on the client anyways.